netlink/conn.go

```diff
@@ -33,8 +33,8 @@ type UEventConn struct {
 // Connect allow to connect to system socket AF_NETLINK with family NETLINK_KOBJECT_UEVENT to
 // catch events about block/char device
 // see:
-// - http://elixir.free-electrons.com/linux/v3.12/source/include/uapi/linux/netlink.h#L23
-// - http://elixir.free-electrons.com/linux/v3.12/source/include/uapi/linux/socket.h#L11
+// - https://elixir.bootlin.com/linux/v6.14.1/source/include/uapi/linux/netlink.h#L23
```

Comment:
It is this link : https://elixir.bootlin.com/linux/v6.14.1/source/include/uapi/linux/netlink.h#L24